### PR TITLE
update katello-agent to include new plugin

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: katello-agent/katello-agent.spec [hunk 1]
```diff
@@ -61,13 +61,15 @@ cp src/katello/agent/katelloplugin.py %{buildroot}/%{_prefix}/lib/gofer/plugins
...
+cp src/yum-plugins/enabled_repos_upload.py %{buildroot}/%{_prefix}/lib/yum-plugins
 
 mkdir -p %{buildroot}/%{_sysconfdir}/yum/pluginconf.d/
 cp etc/yum/pluginconf.d/package_upload.conf %{buildroot}/%{_sysconfdir}/yum/pluginconf.d/package_upload.conf
+cp etc/yum/pluginconf.d/enabled_repos_upload.conf %{buildroot}/%{_sysconfdir}/yum/pluginconf.d/package_upload.conf
 
 mkdir -p %{buildroot}%{_sbindir}
 cp bin/katello-package-upload %{buildroot}%{_sbindir}/katello-package-upload
-
+cp bin/katello-enabled-repos-upload %{buildroot}%{_sbindir}/katello-package-upload
```
Comment (ehelms) [inline katello-agent/katello-agent.spec]: We are overriding the package-upload script?

Comment (jlsherrill) [reply katello-agent/katello-agent.spec]: good catch :)

**Comment by ehelms** [comment]
Looks like you added a binary file

**Comment by jlsherrill** [comment]
Yeah, that was intentional while i'm testing builds.  The katello-agent pr this is meant to be part of has not been merged yet.

**Comment by ehelms** [comment]
You could consider updating the source like this (or separately):

```
%global gittag0 2.10.1

Source0: https://codeload.github.com/Katello/%{name}/tar.gz/%{gittag0}#/%{name}-%{version}.tar.gz

 %prep
%setup -q -n %{name}-%{gittag0}
```

Then you can git annex add the source as "katello-agent-2.10.1.tar.gz"

**Comment by jlsherrill** [comment]
[test]

**Comment by ehelms** [comment]
[test]

**Comment by jlsherrill** [comment]
[test]

**Review by ehelms — APPROVED** [review]

**Pull request merged**